spec/parsers/bmp_parser_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+describe FormatParser::BMPParser do
+  describe 'is able to parse all the examples of BMP' do
+    Dir.glob(fixtures_dir + '/*.bmp').each do |bmp_path|
```

Comment:
Let's move those into the BMP/ directory in fixtures/ - right now it would be too easy to remove the .bmp files from fixtures/ and we wouldn't notice since they are stored there together with all the other examples we imported from FastImage.